### PR TITLE
Map code/setup for changing jobs per map

### PR DESCRIPTION
**Pull request by BlackCrystalic**
## About The Pull Request

Insert in map config "jobs_override" = "#your name in code"

## Why It's Good For The Game
Req feature, idk why

## Changelog

:cl:BlackCrystalic
add: Added easy cfg per map jobs
/:cl:

### CODE REVIEW

**Comment by BlackCrystalic** [comment]
squad momento, bruh

File: code/datums/map_config.dm [hunk 1]
```diff
@@ -117,75 +120,86 @@
...
 			return
 		job_changes = json["job_changes"]
 
+	if("jobs_override" in json)
+		var/jobs_pregen = json["jobs_override"]
+		if(!GLOB.jobs_override_presets[jobs_pregen])
+			log_world("map_config \"jobs_override\" field is invalid!")
+			return
+		for(var/str in GLOB.jobs_override_presets[jobs_pregen])
+		jobs_override = new str
```
Comment (Hekzder) [inline code/datums/map_config.dm]: This doesn't compile

**Comment by BlackCrystalic** [comment]
for some reason i missed that shit, and i know reason... working on 2 thing like to sit on 2 chairs at one time.

**Comment by Koshenko** [comment]
@blackcrystall I'll try to review and test this a bit in the next day or two.  Been a bit busy.  The linter error just appears to be the extraneous `,` in the `mammoth.json` file, so it's nothing important.

**Pull request closed without merging**